docs/migration/5_0.md

```diff
@@ -413,18 +360,11 @@ The following keys have been added to `configurator-common.ts`:
 
 ### QuickOrderService
 
-- Removed `ProductAdapter` from constructor.
-- Added `ProductSearchConnector` to constructor.
 - The `search` method was removed. Use `searchProducts` instead.
 - The `removeEntry` method was removed. Use `softDeleteEntry` instead.
-
+FormErrorsComponent
```

Comment:
remove this :P 

I made similar mistake when searching back and forth haha from here https://github.com/SAP/spartacus/pull/15967#discussion_r917013980

Reply:
Good catch!  Copy paste + search  mistake